root/static/scripts/tests/relationship-editor.js

```diff
@@ -851,7 +851,7 @@ function addRelationships(
 
 function addRelationshipsToRelease(
   rootState: ReleaseRelationshipEditorStateT,
-  source: CoreEntityT,
+  source: RelatableEntityT,
```

Comment:
Commit message says Editable but the changes are to Relatable :)

Reply:
That was commit 7a641bc0585c057ac8283744e3385e0ff50dd398. Now fixed in commit https://github.com/metabrainz/musicbrainz-server/pull/2896/commits/6ba688d993613f0a5a57fb7747f7f8c9e4770a61.